mlir/cmake/modules/AddMLIRPython.cmake

```diff
@@ -228,22 +228,27 @@ endfunction()
 #     aggregate dylib that is linked against.
 function(declare_mlir_python_extension name)
   cmake_parse_arguments(ARG
-    ""
-    "ROOT_DIR;MODULE_NAME;ADD_TO_PARENT"
+    "SUPPORT_LIB"
```

Comment:
I don't feel strongly about this but should we hide this option somehow, e.g., by having it in some `_private_declare_upstream_only_python_extension`? Or is there a case where downstreams can inject support libs? If so, it would make sense to document.